src/osd/ECBackend.cc

```diff
@@ -1762,6 +1763,9 @@ void ECBackend::check_op(Op *op)
     dout(10) << __func__ << " Completing " << *op << dendl;
     writing.pop_front();
     tid_to_op_map.erase(op->tid);
+    bufferlist hbl;
+    op->hoid.encode(hbl);
+    extent_cache.set_evict(hbl.c_str());
```

Comment:
This is not obviously correct.  What ensures that there isn't another write that hasn't completed yet on op->hoid?  How is this supposed to work?